tests/test_opt.py

```diff
@@ -22,15 +22,15 @@
 #
 import unittest
 import numpy
+import logging
```

Comment:
Why  using `logging` here and not replacing all occurrences in `tests`?

Reply:
What other occurrences? The other test files imported `lsst.log` but didn't use it. This test does write log messages so needed a logger.